src/middlewared/middlewared/schema/attribute.py

```diff
@@ -80,7 +80,7 @@ def to_json_schema(self, parent=None):
         """
         raise NotImplementedError("Attribute must implement to_json_schema method")
 
-    def _to_json_schema_common(self, parent):
+    def _to_json_schema_common(self, parent) -> dict[str]:
```

Comment:
`typing.Dict[type_here, value_type_here]`

Reply:
As of Python 3.9 you can use just regular list and dict, but I'll change to the preferred List and Dict

Reply:
I'm also learning the repo as I go, so Dict[str] evaluates to Dict[str, Any] since I wasn't sure